Instead of spewing this over and over, and scrolling useful info
off the screen, just print it once.

--- linux-3.9.0-0.rc3.git0.2.fc19.x86_64/kernel/sched/idle_task.c~	2013-03-21 10:44:17.935087323 -0400
+++ linux-3.9.0-0.rc3.git0.2.fc19.x86_64/kernel/sched/idle_task.c	2013-03-21 10:44:41.936092782 -0400
@@ -35,6 +35,13 @@ static struct task_struct *pick_next_tas
 static void
 dequeue_task_idle(struct rq *rq, struct task_struct *p, int flags)
 {
+	static bool once = 0;
+
+	if (once == 1)
+		return;
+
+	once = 1;
+
 	raw_spin_unlock_irq(&rq->lock);
 	printk(KERN_ERR "bad: scheduling from the idle thread!\n");
 	dump_stack();
